classes/Domain/Services/ResetEmailer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace OpenCFP\Domain\Services;
+
+class ResetEmailer
```

Comment:
Most of what this class is doing is actually cobbling together the message using variables and Twig templates... it also does call the Swift mailer service. It feels like this would end up being a layer of abstraction on top of Swift, both message and mailer service... would that actually gain us anything?